### PR TITLE
Added scale and scroll. Fixed #1633.

### LINKED ISSUE #1633: Add zoom and scroll function for projector controls
like in OpenSlides 1.x:
- zoom up/down
- scroll up/down
- reset zoom & scroll

### PR DESCRIPTION
**Pull request by normanjaeckel**
@emanuelschuetze Please test it.

### CODE REVIEW

File: openslides/core/views.py [hunk 1]
```diff
@@ -284,6 +299,50 @@ def clear_elements(self, request, pk):
...
+        Example:
+
+        {
+            "action": "scale",
+            "direction": "up"
+        }
+        """
+        if not isinstance(request.data, dict):
+            raise ValidationError({'data': 'Data must be a dictionary.'})
+        if request.data.get('action') not in ('scale', 'scroll') or request.data.get('direction') not in ('up', 'down'):
```
Comment (ostcar) [inline openslides/core/views.py]: Can you shorten this line please?

**Comment by ostcar** [comment]
One comment, then you can merge it (when @emanuelschuetze has tested it...)

File: openslides/core/views.py (hunk 1 above)
Comment (normanjaeckel) [inline openslides/core/views.py]: Done.

**Comment by normanjaeckel** [comment]
@emanuelschuetze Please test it and merge if at your will.

**Comment by emanuelschuetze** [comment]
@normanjaeckel If I try to send this post request
` $http.post('/rest/core/projector/1/control_view/', {"action": "scale", "direction": "up"});`
I get the error `{"detail":"Sie sind nicht berechtigt, diese Aktion durchzuführen."}`. Any idea?

**Comment by emanuelschuetze** [comment]
I found the bug in views.py:
```
--- a/openslides/core/views.py
+++ b/openslides/core/views.py
@@ -153,7 +153,7 @@ class ProjectorViewSet(ReadOnlyModelViewSet):
         if self.action in ('metadata', 'list', 'retrieve'):
             result = self.request.user.has_perm('core.can_see_projector')
         elif self.action in ('activate_elements', 'prune_elements', 'update_elements',
-                             'deactivate_elements', 'clear_elements'):
+                             'deactivate_elements', 'clear_elements', 'control_view'):
```
I'll fix it in my commit.

**Comment by normanjaeckel** [comment]
Sorry. Tell me if everything is working. Then I will rebase all on top of master and merge the commit and also the fix. After this you can rebase your work on top of new master.

**Comment by emanuelschuetze** [comment]
One last wish: I need a possibility to **reset** the scale and/or scroll value to 0. Currently there are +/-1 steps only. See 1.7 demo.

**Comment by emanuelschuetze** [comment]
BTW: The rest works nice.

**Comment by ostcar** [comment]
The changed commit is ok for me (but not for travis...)

**Pull request merged**